### PR TITLE
EMBCESS-900:  viewmodels and mappings

### PR DESCRIPTION
**Pull request by ytqsl**

### CODE REVIEW

File: embc-app/Controllers/VolunteersController.cs [hunk 1]
```diff
@@ -50,7 +50,7 @@ public async Task<IActionResult> Get(string id)
         [HttpPost]
         public async Task<IActionResult> Create([FromBody] Volunteer item)
         {
-            var existing = dataInterface.GetVolunteerByExternalId(item.Externaluseridentifier);
```
Comment (ytqsl) [inline embc-app/Controllers/VolunteersController.cs]: no idea why this is included in this commit..

Comment (asanchezr) [reply embc-app/Controllers/VolunteersController.cs]: @ytqsl that commit above is to deal with duplicate bceid usernames when creating ESS users

Comment (ytqsl) [reply embc-app/Controllers/VolunteersController.cs]: I know, but it is not part of this story, git included it in the diff

Comment (asanchezr) [reply embc-app/Controllers/VolunteersController.cs]: yeah, weird. Did you rebase?

Comment (CodingVelocista) [reply embc-app/Controllers/VolunteersController.cs]: It's not the right code either?

Comment (ytqsl) [reply embc-app/Controllers/VolunteersController.cs]: I'll rebase again and update

**Pull request merged**